src/styles/application/_errorScreen.scss

```diff
@@ -0,0 +1,42 @@
+.integr8ly-errorscreen {
+  &-backdrop {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    background-color: $pf-color-blue-600;
+    background-image: url("../../img/provisioning-bg.png");
+    z-index: 5;
+  }
+
+  &-error-img {
+    margin-top: 15%;
+    margin-bottom: -20%;
+  }
+
+  &-logo {
+    background-size: cover;
+    height: 36px;
+    width: 113px;
+    position: absolute;
+    top: 85px;
+    right: 50px;
```

Comment:
This should be `right: 25px;` in order to match the positioning on the loading screens - all placements of the Red Hat logo should be the same.